lab-jeff/README.md

```diff
@@ -0,0 +1,30 @@
+![Triforce](./resources/triforce2.gif)
+
+#HTTP Server#
+
+###Summary###
+An HTTP server with a Cowsay() API endpoint that takes GET and POST requests and responds with a cow!
+
+###Instructions###
+From the root directory of 'lab-jeff' run the following command in your terminal:
+
+`npm install`
+
+Then to start the server:
+
+`node ./api/cowsay/server.js`
```

Comment:
You don't actually need to create a directory structure to match your route. Your server.js file should live in your project root.